README.md

```diff
@@ -1,92 +1,122 @@
-# OOP school library
+# Enumerable-project
 
-> In this project, I will start building my school library app. I'm practicing design patterns for building OOP applications.
-## Built With
+## Getting Started
```

Comment:
- [ ] Please remove this section, because you already have added it below after the `key-feature` section. Leaving this section can make someone confuse and also make your documentation looks unprofessional.